sqlite3.go

```diff
@@ -81,8 +82,25 @@ _sqlite3_exec(sqlite3* db, const char* pcmd, long long* rowid, long long* change
 static int
 _sqlite3_step(sqlite3_stmt* stmt, long long* rowid, long long* changes)
 {
-  int rv = sqlite3_step(stmt);
+  extern int unlock_notify_wait(sqlite3 *db);
+  int rv;
   sqlite3* db = sqlite3_db_handle(stmt);
+
+  for (;;) {
```

Comment:
Shouldn't this be
```go
for {
```

Reply:
I don't have access to the dev environment right now but this `for` looks like C code.